src/helm/azure-pipelines-agent/templates/extraManifests.yaml

```diff
@@ -0,0 +1,4 @@
+{{ range .Values.extraManifests }}
+---
+{{ tpl (toYaml .) $ }}
```

Comment:
Enabling the YAML with `tpl`, current Helm variables will be available, incl utilities one stored in `_helpers.tpl`. The risk is that as there is no API contract with those utilities functions, they can be updated (refacto, security, fix, etc), thus the compatibility with this will break time to time. Seeing the maintenance cost, I don't see the added value there. Can you describe what you got in mind?

Here is a proposal with a simple YAML format check:

```tpl
{{- toYaml . }}
```

Reply:
You're correct, `tpl` is "a bit" overkill for this anyway, changed to rely only on `toYaml`